Python/ceval.c

```diff
@@ -3716,23 +3716,25 @@ too_many_positional(PyCodeObject *co, Py_ssize_t given, Py_ssize_t defcount,
     Py_ssize_t kwonly_given = 0;
     Py_ssize_t i;
     PyObject *sig, *kwonly_sig;
+    Py_ssize_t co_posonlyargcount = co->co_posonlyargcount;
     Py_ssize_t co_argcount = co->co_argcount;
+    Py_ssize_t co_total_positional = co_argcount + co_posonlyargcount;
```

Comment:
Since it is not a direct copy of the code object, can you remove co_ prefix from the variable name?